src/discussions/discussions-home/DiscussionSidebar.test.jsx

```diff
@@ -85,7 +84,7 @@ describe('DiscussionSidebar', () => {
         },
       })]);
     renderComponent();
-    await act(async () => expect(await screen.findAllByText('Thread by other users')).toBeTruthy());
+    await waitFor(async () => expect(await screen.findAllByText('Thread by other users')).toBeTruthy());
```

Comment:
This can be simplified to

```suggestion
    await screen.findAllByText('Thread by other users');
```

because `findAllByText` will throw if it doesn't find any matches ([docs](https://testing-library.com/docs/react-testing-library/cheatsheet/))